src/WorldStorage/EnchantmentSerializer.cpp

```diff
@@ -14,8 +14,8 @@ void EnchantmentSerializer::WriteToNBTCompound(const cEnchantments & a_Enchantme
 	for (cEnchantments::cMap::const_iterator itr = a_Enchantments.m_Enchantments.begin(), end = a_Enchantments.m_Enchantments.end(); itr != end; ++itr)
 	{
 		a_Writer.BeginCompound("");
-			a_Writer.AddShort("id",  itr->first);
-			a_Writer.AddShort("lvl", itr->second);
+			a_Writer.AddShort("id",  static_cast<Int16>(itr->first));
+			a_Writer.AddShort("lvl", static_cast<Int16>(itr->second));
```

Comment:
What happens if the enchantment level is larger than `Int16::max`? Shouldn't we `Clamp()`?